src/Backoffice/Airlines/App/Controllers/ListAirlineController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Airlines\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Backoffice\Airlines\App\Requests\FilterAirlinesRequest;
+use Lightit\Backoffice\Airlines\Domain\Actions\ListAirlineAction;
+
+class ListAirlineController
+{
+    public function __invoke(ListAirlineAction $listAirlineAction, FilterAirlinesRequest $filters): JsonResponse
+    {
+        $listAirlineAction->execute($filters->toDto());
```

Comment:
```suggestion
    public function __invoke(ListAirlineAction $listAirlineAction, FilterAirlinesRequest $filterAirlinesRequest): JsonResponse
    {
        $listAirlineAction->execute($filterAirlinesRequest->getFilters());
```
I'm just renaming things, I see it clearer that way

Reply:
Request was no longer needed because of spatie, so variable no longer exists